lsp-lens.el

```diff
@@ -407,8 +408,8 @@ CALLBACK - callback for the lenses."
                      (lambda ()
                        (--map (overlay-put it 'before-string
                                            (overlay-get it 'lsp-original))
-                              lsp-lens--overlays))))))
-      (when action (funcall-interactively action)))))
+                              lsp-lens--overlays)))))
+      (when actions (funcall-interactively (cl-third actions))))))
```

Comment:
I don't understand how does this fixes avy returning `t`? We will still call cl-third over `t`?

Reply:
This is an unrelated change, sorry. `avy-process` returns t only if there are no input lenses. This is handled by the `unless lsp-lens--overlays` `user-error` at the top. If there are lens overlays, `avy-process` cannot ever return t here. Should I revert?

Reply:
If this is unrelated it seems like it is better to revert or use better naming since it does not return `actions`.

Reply:
I decided to revert, because micro-optimizing `cl-third` isn't worth-it and because coming up with a name that is as good as `action` here turned out to be a challenge (`result`?, `data`?). In addition, this won't error if the selected lens has no action (maybe we'll want to support selecting actionless lenses in the future for some reason?).